core/src/data/value/into.rs

```diff
@@ -436,6 +440,7 @@ mod tests {
         test!(Value::I8(122), Ok(122));
         test!(Value::I64(122), Ok(122));
         test!(Value::F64(122.0), Ok(122));
+        test!(Value::F64(122.1), Ok(122));
```

Comment:
```rust
        test!(Value::F64(122.0), Ok(122));
        test!(Value::F64(122.1), Ok(122));
```

If it is the same case, it seems that there is no need to add it.

Reply:
I will change to test from x.1 to x.9.  that example will check for round vs truncate.